api-specs/gem.proto

```diff
@@ -24,12 +24,11 @@ message Gem {
 }
 
 message DropRequest {
-  string message = 1;
+  Gem gem = 1;
```

Comment:
Not entirely sure, but does this work if the gem's id is not present? Because the id will only be generated after the drop request is processed

Reply:
jk I think what you have works!